integrationtests/as-integration-embedded/src/test/java/org/infinispan/test/integration/as/InfinispanStoreJpaIT.java

```diff
@@ -64,4 +69,21 @@ public void testCacheManager() {
       cm.stop();
    }
 
+   @Test
+   public void testXmlConfig() throws IOException {
+      EmbeddedCacheManager cm = new DefaultCacheManager("jpa-config.xml");
+      Cache<String, KeyValueEntity> specificCache = cm.getCache("specificCache");
+      validateConfig(specificCache);
+      KeyValueEntity entity = new KeyValueEntity("k", "v");
+      specificCache.put(entity.getK(), entity);
+      cm.stop();
```

Comment:
`stop()` should be in a finally block, or you could use `TestingUtil.withCacheManager()`.